src/css/components/_keyboard.scss

```
@@ -121,20 +127,26 @@
     align-items: center;
     margin-left: -27px;
     color: #777777;
+    background-color: color(white);
 }
 
 .light-theme .keyboard .mask-option-label {
     margin-left: 5px;
+    line-height: 1;
+    margin-bottom: 10px;
 }
 
 .keyboard .input-row {
     width: 100%;
     white-space: nowrap;
+    display: flex;
+    align-items: end;
```

```suggestion
    align-items: center;
```

on master this element is vertical centered